python/py-kombine/Portfile

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-kombine
+version             0.8.3
+
+categories-append   science
+maintainers         {uoregon.edu:bfarr @bfarr} openmaintainer
+
+platforms           darwin
+license             GPL-3
+
+description         An embarrassingly parallel, kernel-density-based \
+                    ensemble sampler
+long_description    kombine is an ensemble sampler that uses a clustered \
+                    kernel-density-estimate proposal density, allowing for \
+                    massive parallelization and efficient sampling.
+homepage            https://github.com/bfarr/kombine
+
+master_sites        pypi:k/kombine
+distname            kombine-${version}
+checksums   rmd160  74c1ba9d7f4f1866f79f4628c868943afb9fabec \
+            sha256  da7a9542600c81df4b2ede2772b730dacef61ea3bc19b495f4fa445ff2b92f75 \
+            size    17495
+
+python.versions     27 34 35 36
+python.default_version 27
```

Comment:
Should we be defaulting to Python 3 here? (I know too little about the port machinery for python packages.)

Reply:
@pmetzger, this default is chosen to match the system version of python on macOS. @bfarr can comment on whether he wants python3 as default.

Reply:
@mf2k

Reply:
Honestly, the default version is not very important. It should not hold up this port. Whatever it is currently is fine.

Reply:
Gotcha. So at this point what we're waiting on is cleanup of the `python.versions` line and nothing else.